armi/reactor/tests/test_reactors.py

```diff
@@ -834,8 +834,6 @@ def test_getDominantMaterial(self):
         self.assertEqual(dominantFuel.getName(), "UZr")
         self.assertEqual(dominantCool.getName(), "Sodium")
 
-        self.assertEqual(list(dominantCool.getNuclides()), ["NA23"])
```

Comment:
Why was this line deleted? Is it related to the main purpose of this PR, or is it in here by accident?

Reply:
That is part of another PR now, I removed that line from this one.

Ignore it.

(It was because of a `DeprecationWarning`.)